deps/wazuh_testing/wazuh_testing/tools/wazuh_manager.py

```diff
@@ -32,11 +38,30 @@ def remove_agents(agents_id, remove_type='wazuhdb'):
                 'older_than': '0s'
             }
             url = f"{api_details['base_url']}/agents"
-            response = requests.delete(url, headers=api_details['auth_headers'], params=payload, verify=False)
+            response = requests.delete(
+                url, headers=api_details['auth_headers'], params=payload, verify=False)
             response_data = response.json()
             if response.status_code != 200:
                 raise RuntimeError(f"Error deleting an agent: {response_data}")
 
 
 def remove_all_agents(remove_type):
     remove_agents(list_agents_ids(), remove_type)
+
+
+@retry(AttributeError, attempts=10, delay=5, delay_multiplier=1)
+def wait_agents_active_by_name(agents_names: List[str]):
+    """Wait until all agents received in the agents_name list are active.
+
+    Args:
+        agents_names (list[str]): The list with agents names to check.
+    Raises:
+        AttributeError: If any agent is not active. Combined with the retry
+            decorator makes a wait loop until all the agents are active.
+    """
+    for name in agents_names:
+        name = name.replace("\r", "").replace("\n", "")
+        command = f'{WAZUH_PATH}/bin/agent_control -l | grep "{name}"'
```

Comment:
Whenever It contains a variable inside, you should use `"` else `'`.

Reply:
Fixed on commit [`a915da5`](https://github.com/wazuh/wazuh-qa/pull/3662/commits/a915da5cf02906ea8e70673489fb69f3bde08639)